src/util/duration.cpp

```diff
@@ -48,5 +48,36 @@ QString DurationBase::formatSeconds(double dSeconds, Precision precision) {
 
     return durationString;
 }
+// static
+QString DurationBase::formatKiloSeconds(double dSeconds, Precision precision) {
+    if (dSeconds < 0.0) {
+        // negative durations are not supported
+        return "?";
+    }
+
+    const qint64 days = static_cast<qint64>(std::floor(dSeconds)) / kSecondsPerDay;
+    dSeconds -= days * kSecondsPerDay;
+
+    int kilos = (int)dSeconds / 1000;
+    double seconds = floor(fmod(dSeconds, 1000));
+    double subs = fmod(dSeconds, 1);
+
+    QString durationString =
+            (days > 0 ? (QString::number(days) %
+                         QLatin1String("d, ")) : QString()) %
```

Comment:
Why do we need to deal with days? Can't we just count seconds?

Reply:
I think we schould remove the day part entirely. No track will have this length, and if there is what we have unlikely the screen space for this.
I personally would prefer to see 
30:10:01,86 instead of 1d,6:10:01,86